feat: replacing polling for lobby updates with LobbyEvents [MTT-5425] (#805)
Unity-Technologies/com.unity.multiplayer.samples.coop: CHANGELOG.md

Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).

## [unreleased] - yyyy-mm-dd

### Changed

> * Replaced our polling to obtain updates to the lobby that a client or host has joined with a subscrition to LobbyEvents (#805). This saves up a significant amount of bandwidth usage to and from the service, since updates are infrequent in this game. Now clients and hosts only use up bandwidth on the Lobby service when it is needed.

Reviewer: ```suggestion
* Replaced our polling for lobby updates with a subscription to the new Websocket based LobbyEvents (#805). This saves up a significant amount of bandwidth usage to and from the service, since updates are infrequent in this game. Now clients and hosts only use up bandwidth on the Lobby service when it is needed.
```

Reviewer: I looked at your back of the napkin calculations in the jira comments. It'd be great to add a summary here.
Quick note, I don't think you need to go as far as calculating game length. Just going with user count, all doing requests every 2 seconds would be enough, no need to calculate number of games. You don't care whether they are the same users, just that on average you'll have x amount of users making poll requests every 2 seconds.
I'd validate my calculations in the lobby channels, just to make sure we didn't miss anything before sharing an actual $ amount.
So 725B * 30 * 100k CCU = 2.175GB per minute. Or 93TB per month.

Author: Good point! I'll add a streamlined version of these calculations here
